app/routers/api/settings.py

```
@@ -50,3 +50,48 @@ def update_smtp_config(
         smtp_use_tls=smtp_data.smtp_use_tls if smtp_data.smtp_use_tls is not None else True,
     )
     return {"message": "SMTP configuration updated successfully"}
+
+
+@router.put("/oidc")
+def update_oidc_config(
+    oidc_data: OIDCSettingsUpdate,
+    db: Annotated[Session, Depends(get_db)],
+    current_user: Annotated[dict, Depends(PermissionChecker("notifiers"))],
```

Ruff (0.15.20)

[warning] Unused function argument: `current_user`

(ARG001)